the_hack_series/resources/FACILITATION.md

```diff
@@ -0,0 +1,135 @@
+
+- Be a good host by making every participant feel like you were welcoming them like the president!
+- It is your job to make sure that everyone talks to everyone else. People will naturally not talk to each other!
+
+**Roles:**
+
+1. **The Front Door Greeter:**
+  - Stands by the front door and warmly greets people as they come
+  - Gives everyone a sticker
+2. **The Guide**
+  - Walks hackers from the front door to the hacker space
+  - Offers guests water
+  - Gives them a brief tour of the Make School and explains briefly what it is
+3. **The Conversationalist**
+  - Stays in the hacker space and makes sure that there is _always_
+    conversation happening between hackers
+  - When a new hacker comes up, the conversationalist introduces them to
+    someone already there
+
+### 1:10  : Hack Series Introduction & Icebreakers
+
+1. Excitedly welcome everyone to the Hack Series
+2. Play the alliterative name game icebreaker standing up.
+3. Explain groups and have people count off into groups with a max group size of 4.
+  - Emphasize that groups are meant to help each other but not for people to
+    be working on the same project together.
+4. Introduce the schedule for the day (note that the schedule should already be
+   projected on the screen)
+
+### 1:20 : Introduce Part I
+
+1. Introduce the Portfolio Workshop.
+  1. Demo the <a href="http://jsbin.com/gist/81d45193dab5236afbba?output" target="_blank_">Final Product</a>
+  2. Show other [examples of what people made](websites/README.md)
+  3. Introduce that they can Google for how to do something
+2. If you have more experience, feel free to work on your own projects!
+3. Set everyone up on http://tlk.io/hackseries
+  - Send out the https://bit.ly/hackserieshome which links them to the workshop
+
+### 1:25 : Part I: Follow the step-by-step workshop
+
+1. Circle around and make sure everyone has portfolio workshop open
+2. Make sure everyone is signed in to JS Bin
+3. Instead of asking "How are you doing?" or "Do you need help?" ask "What's
+   your progress so far?" or "What do you want to do next?"
+
+### 2:30 : Pizza Time
+
+Facilitate conversation between hackers.
+
+### 3:00  : Introduce Part II
+
+- Explain that this time is for hackers to customize their website.
+- Feel free to share ideas with your partners
+- Demo [other hackers' websites](EXTENSION.md)
+- Explain how demos work
+
+### 3:05  : Part II: Customize your website
+
+1. Go around and check in with people for what ideas they're working on
+2. Instead of asking "How are you doing?" or "Do you need help?" ask "What's
+   your progress so far?" or "What do you want to do next?"
+
+### 3:40 : Collect Demo Links in tlk.io
+
+Do it.
+
+### 3:45  : Demos!
+
+Open up every tab of the submitted demo links and that will be the presentation
+order.
+
+Encourage everyone to present.
+
+### 3:55  : Finish
+
+Thank everyone for coming.
+
+# Feedback
+
+- Explain and **_HEAVILY EMPHSIZE** the importance of feedback because it helps
```

Comment:
Should this be `**_HEAVILY EMPHASIZE_**` instead? It's currently misspelled and there's no closing `_`.